stdlib/importlib/machinery.pyi

```diff
@@ -123,10 +123,10 @@ def all_suffixes() -> list[str]: ...
 
 class FileFinder(importlib.abc.PathEntryFinder):
     path: str
-    def __init__(self, path: str, *loader_details: tuple[Type[importlib.abc.Loader], list[str]]) -> None: ...
+    def __init__(self, path: str, *loader_details: tuple[typing.Type[importlib.abc.Loader], list[str]]) -> None: ...
```

Comment:
```suggestion
    def __init__(self, path: str, *loader_details: tuple[type[importlib.abc.Loader], list[str]]) -> None: ...
```

Reply:
Right, that explains it.  The flake8 hint wasn't clear on it's own, especially since nothing in the code uses `builtin.type`   _anywhere_ and the only places that seem to use it use `from builtins import type as Type`  which is what I did in my last change here.